src/vfShow.ts

```diff
@@ -1129,26 +1134,18 @@ export class Renderer {
      *
      * Also sets s.storedVexflowStave to stave.
      */
-    applyFormatterInformationToNotes(
-        stave: Vex.Flow.Stave,
-        s?: stream.Stream,
-        formatter?: Vex.Flow.Formatter,
-    ) {
+    applyFormatterInformationToNotes(stave: Stave, s?: stream.Stream, formatter?: Formatter) {
         if (s === undefined) {
             s = this.stream;
         }
         const sClef = s.getSpecialContext('clef') || s.getContextByClass('Clef') || _clefSingleton;
         let noteOffsetLeft = 0;
         // var staveHeight = 80;
         if (stave !== undefined) {
-            noteOffsetLeft = stave.start_x + stave.glyph_start_x;
+            const noteStartX = stave.getNoteStartX();
+            noteOffsetLeft = noteStartX; // UNKNOWN: + stave.glyph_start_x;
```

Comment:
This was an important fix at some point -- I can't remember which setting had the glyphs start at a different x than before, but I remember it being a lot of trial and error to find this one.  I might look at the "blame" to figure out when it came in.